tests/test_wog.py

```diff
@@ -456,3 +456,37 @@ def test_wisps_of_the_old_gods():
 	for wisp in game.player1.field:
 		assert wisp.atk ==  wisp.health == 3
 		assert wisp.id == "OG_195c"
+
+
+def test_xaril_poisoned_mind():
+	game = prepare_empty_game()
+	xaril = game.player1.give("OG_080")
+	xaril.play()
+	assert game.player1.hand[0] in ("OG_080b", "OG_080c", "OG_080d", "OG_080e", "OG_080f")
+	game.player1.give("DS1_185").play(target=xaril)
+	assert game.player1.hand[1] in ("OG_080b", "OG_080c", "OG_080d", "OG_080e", "OG_080f")
+
+
+def test_xaril_toxins():
```

Comment:
You're not actually testing any of the toxins though.

Also if you do, they should just have their own test. But it's notnecessary.